packages/application-to-register-webapp/src/views/land/add-legal-agreement-parties.html

```diff
@@ -83,7 +83,6 @@ <h1 class="govuk-heading-l" tabindex="-1">
                                     name: "organisation[" + i + "][role]",
                                     value: "County Council",
                                     text: "County Council",
```

Comment:
To be consistent with the ID updates on other inputs this ID should be updated to align with county council